plugins/elantp/fu-elantp-firmware.c

```diff
@@ -72,6 +108,30 @@ fu_elantp_firmware_check_magic(FuFirmware *firmware, GBytes *fw, gsize offset, G
 		}
 	}
 
+	if (self->force_table_addr == 0)
+		return TRUE;
+
+	for (gsize i = 0; i < sizeof(elantp_signature); i++) {
```

Comment:
I think I'd rather this loop was in the `if` block in case we have other signatures in the future.